packages/cysync-core/src/pages/OnBoarding/OnboardingPageLayout.tsx

```diff
@@ -78,6 +80,7 @@ export const OnboardingPageLayout: React.FC<OnboardingPageLayoutProps> = ({
       text={text}
       currentState={currentState}
       totalState={totalState}
+      isDialogOpen={isDialogOpen}
```

Comment:
Why is there a dialog inside layout?

Reply:
Changed the flag to `showBlurBackground`